File: java/Association/Processor.java
 
public class Processor
{
    private double clockSpeed;
    
    public Processor(){
    	clockSpeed =2.5;
    	}	
    	
    public Processor(double cs){
    	       clockSpeed=cs;
        }
        
   public double getClockSpeed(){
   		return clockSpeed;
   }
   
   public void setClockSpeed(double cs){
   		clockSpeed=cs;
   }
	
}

File: cpp/Video 05-02 (JAVA CS)Composition Aggregation Assoication/CS/OOP/Association/Processor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Association
{
    class Processor
    {
        private double clockSpeed;

        public double ClockSpeed
        {
            get { return clockSpeed; }
            set { clockSpeed = value; }
        }
        public Processor()
        {
            ClockSpeed = 2.5;
        }
        public Processor(double cs)
        {
            clockSpeed = cs;
        }
      
    }
}
